dojo/tools/wazuh/parser.py

```diff
@@ -18,73 +25,59 @@ def get_label_for_scan_types(self, scan_type):
     def get_description_for_scan_types(self, scan_type):
         return "Wazuh"
 
-    def get_findings(self, filename, test):
-        data = json.load(filename)
+    def get_findings(self, file, test):
+        data = json.load(file)
+
+        if not data:
+            return []
+
         # Detect duplications
         dupes = dict()
 
-        try:
-            vulnerability = data[next(iter(data.keys()))]["affected_items"]
-        except (KeyError, StopIteration):
-            return list()
-
-        if vulnerability is None:
-            return list()
+        # Loop through each element in the list
+        for entry in data:
+            vulnerabilities = entry.get("data", {}).get("affected_items", [])
+            for item in vulnerabilities:
+                if (
+                    item["condition"] != "Package unfixed"
+                    and item["severity"] != "Untriaged"
+                ):
+                    id = item.get("cve")
+                    package_name = item.get("name")
+                    package_version = item.get("version")
+                    description = item.get("condition")
+                    severity = item.get("severity").capitalize()
+                    agent_ip = item.get("agent_ip")
+                    links = item.get("external_references")
 
-        for item in vulnerability:
-            if (
-                item["condition"] != "Package unfixed"
-                and item["severity"] != "Untriaged"
-            ):
-                id = item.get("cve")
-                package_name = item.get("name")
-                package_version = item.get("version")
-                description = item.get("condition")
-                if item.get("severity") == "Untriaged":
-                    severity = "Info"
-                else:
-                    severity = item.get("severity")
-                if item.get("status") == "VALID":
-                    active = True
-                else:
-                    active = False
-                links = item.get("external_references")
-                title = (
-                    item.get("title") + " (version: " + package_version + ")"
-                )
-                severity = item.get("severity", "info").capitalize()
-                if links:
-                    references = ""
-                    for link in links:
-                        references += f"{link}\n"
-                else:
-                    references = None
+                    if links:
+                        references = "\n".join(links)
+                    else:
+                        references = None
 
-                if id and id.startswith("CVE"):
-                    vulnerability_id = id
-                else:
-                    vulnerability_id = None
+                    title = item.get("title") + " (version: " + package_version + ")"
+                    dupe_key = title
```

Comment:
`dupe_key = title` does not work very well. Adding the agent_ip solved the problem for me. 
`dupe_key = title+agent_ip`